### ISSUE TITLE
KB: Add and Filter buttons should be separate in tab order from project tree

### ISSUE THREAD

**Issue by beyackle** [microsoft/BotFramework-Composer#6460]
## Describe the bug

Keyboard navigation should see the project tree as a single component that can be navigated using arrow keys, but currently it also includes the Add and Filter buttons on top, which aren't really part of the tree.

## Version

1.3.1

## Expected behavior

Add and Filter buttons should be in the tab order independently, not part of the Project Tree.
